wp-content/themes/pawar2018/front-page.php

```diff
@@ -1,188 +1,164 @@
-              <p>
-                Pass criminal justice reform and refocus resources from prisons to diversion programs, job training and placement, and social and mental health services.
-              </p>
+          <?php
+            $args = array(
+            'post_type' => 'pillars',
+            'orderby' => 'meta_value',
+            'order' => 'ASC') ?>
+          <?php $loop = new WP_Query($args); ?>
+          <?php while ( $loop->have_posts() ) : $loop->the_post(); ?>
+            <div class="column column-block pillar-content">
+              <img src="<?php echo get_field('pillar_icon') ?>" class="issue-icon" alt="<?php echo get_field('pillar_alt') ?>">
+              <div class="pillar-copy">
+                <h4 class="pillar-header"><?php the_title();?></h4>
+                <p><?php the_content() ?></p>
+              </div>
             </div>
-          </div>
+          <?php endwhile; ?>
+            <?php wp_reset_postdata(); ?>
         </div>
         <div class="pillar-button">
-          <a href="<?php echo esc_url( home_url( '/issues' )) ?>" class="button">
-            See the Issues
+          <a href="<?php the_field('pillar_button_link'); ?>" class="button">
+            <?php the_field('pillar_button_text'); ?>
           </a>
         </div>
       </div>
       <div class="event-wrapper">
         <div class="event-content">
           <h4 class="section-title">
-            Upcoming Events
+            <?php the_field('event_headline'); ?>
           </h4>
-          <h1 class="main-callout">Let's talk.</h1>
+          <h1 class="main-callout"><?php the_field('event_subheadline'); ?></h1>
           <div class="row small-collapse">
-             <?php
-                $today = date('Ymd');
-                $args = array(
-                'post_type' => 'events',
-                'posts_per_page' => 2,
-                'meta_key' => 'start_date',
-                'orderby' => 'meta_value_num',
-                'order' => 'ASC',
-                'meta_query'  => array(
-                      array(
-                          'key' => 'start_date',
-                          'type' => 'NUMERIC',
-                          'value' => $today,
-                          'compare' => '>=',
-                          )
-                      ),
-                  );
 
-              ?>
-              <?php $loop = new WP_Query( $args ); ?>
-              <?php while ( $loop->have_posts() ) : $loop->the_post(); ?>
-              <div class="column small-11 large-8 event-copy">
-                <?php if (get_field('link')) : ?>
-                <a href="<?php the_field('link'); ?>">
-                <?php endif; ?>
-                    <p class="event-date">
-                        <?php the_field('start_date'); ?> at <?php the_field('start_time'); ?>
-                    </p>
-                    <h5 class="event-title">
+            <?php $events = new Eventbrite_Query(
```

Comment:
We can make this a little better for local dev by wrapping the entire thing in the following:

```

<?php
// eventbrite doesn't work locally :(
if (class_exists(Eventbrite_Query)) : ?>
    // all the stuff you have already
    wp_reset_postdata(); // you already have this - just repeating for placement
else :
    // maybe something more informative here?
    get_template_part('content', 'none');
endif;
?>
```

That will at least suppress the error message and allow the rest of the page to render.